src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/SubjectAccessRequestService.kt

```diff
@@ -48,7 +51,10 @@ class SubjectAccessRequestService(
     return HmppsSubjectAccessRequestContent(
       content = SubjectAccessRequestContent(
         induction = induction?.let { inductionMapper.toInductionResponse(induction) },
-        goals = goals.map { goalMapper.fromDomainToModel(it) }.toSet(),
+        goals = goals.map {
```

Comment:
This is the area that this ticket is really trying to address.
Previously the SAR endpoint returned goals, but those goals did not have any goalNotes attached to them (because it was using the previous goalMapper that did not take the goalNotes as well)
Now it does 👍